setup.py

```diff
@@ -1,18 +1,12 @@
-# -*- coding: utf-8 -*-
-
-# Learn more: https://github.com/Ensembl/ols-client
 import os
 
 from setuptools import setup, find_packages
 
-with open(os.path.join(os.path.dirname(__file__), 'README.md')) as f:
-    readme = f.read()
-
-with open(os.path.join(os.path.dirname(__file__), 'LICENSE')) as f:
-    license_ct = f.read()
+with open('README.md', 'r', encoding='utf-8') as f:
```

Comment:
The previous statement was supposed to allow build from a random directory, was it related to the twine break? 
Did you try to run the pip install from your tar.gz to validate that it's still ok?

Reply:
It wasn't related. But I think building the path like that would be needed if one wanted to install from the source code (not the package) hence executing `setup.py`. But n that case I think one would run `pip install` from the same directory that contains `setup.py`.